Source/QuantityTypes.Tests/Quantities/AngleTests.cs

```diff
@@ -54,5 +54,15 @@ public void ToString_NorwegianCulture_CheckSpaces()
             var a = 90 * Angle.Degree;
             Assert.AreEqual("90,0°", a.ToString("0.0 [°]", CultureInfos.Norwegian));
         }
+
+        [Test]
+        public void ParseStringToAngleType()

```

Comment:
Do we need to use reflection in the test? What do you want to test - getting the correct parse method by reflection, or the parse method itself?